### PR TITLE
Update CHANGELOG 1.8.5

### PR DESCRIPTION
**Pull request by nywilken**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,73 @@
 ## 1.8.5 (Upcoming)
 
+### NOTES:
+* data/packer-image-iteration has been removed. This was an undocumented and
+     unusable data source that was built for experimentation but not release. It
```
Comment (nywilken) [inline CHANGELOG.md:5]:
```suggestion
     unusable data source that was built for experimentation but not released. It
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+     should not affect users in any way but is being mentioned for visibility
+     into the changes within this release.
```
Comment (nywilken) [inline CHANGELOG.md:7]:
```suggestion
     should not affect users in any way but is being mentioned for visibility
     purposes,  as changes within this release.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,73 @@
...
+     [GH-12111](https://github.com/hashicorp/packer/pull/12111)
+
+### FEATURES:
+* core: Metadata for template types such as JSON or HCL2 has been added to the
```
Comment (nywilken) [inline CHANGELOG.md:11]:
```suggestion
* core: Metadata for  capturing template types such as JSON or HCL2 has been added to the
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,73 @@
...
+     HCP Packer registry metadata. Upon running a `packer build` type of
```
Comment (nywilken) [inline CHANGELOG.md:12]:
```suggestion
     HCP Packer registry metadata. Upon running a `packer build` the type of
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,73 @@
...
+     template used during execution will be sent along to the registry as
+     additional build
+     metadata.[GH-1232](https://github.com/hashicorp/packer/pull/12132)
```
Comment (nywilken) [inline CHANGELOG.md:15]:
```suggestion
     additional build metadata.[GH-1232](https://github.com/hashicorp/packer/pull/12132)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,73 @@
...
+     [GH-12152](https://github.com/hashicorp/packer/pull/12152)
+* core/hcl2: Variable definition files containing unreferenced variable within
+     an HCL2 template will no longer warn during build execution. Warnings will
+     be displayed to a user during template validation, which can be disabled by
+     passing the `-no-warn-undeclared-var` flag to the validate command.
+     [GH-12104](https://github.com/hashicorp/packer/pull/12104)
+     [GH-12109](https://github.com/hashicorp/packer/pull/12109)
+* core: Docker images have been updated to include the `xorriso` package for
+     supporting the creation of ISO
+     files.[GH-12081](https://github.com/hashicorp/packer/pull/12081)
```
Comment (nywilken) [inline CHANGELOG.md:44]:
```suggestion
     supporting the creation of ISO files.[GH-12081](https://github.com/hashicorp/packer/pull/12081)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,73 @@
...
+* core: Bump Go version to 1.18.9 to address vulnerability GO-2022-1144, which
+     concerns the net/http and golang.org/x/net
+     packages.[GH-12153](https://github.com/hashicorp/packer/pull/12153)
+* core: Fix the registration of deleted input artifacts in HCP Packer when
+     setting `keep_input_artifacts` to
+     false.[GH-11462](https://github.com/hashicorp/packer/pull/11967)
+* core: To mitigate against the creation of incomplete HCP Packer iterations
+     the ability to switch between template types during the building of a
+     single iteration will now trigger a runtime build error.
+     [GH-1232](https://github.com/hashicorp/packer/pull/12132)
```
Comment (nywilken) [inline CHANGELOG.md:69]:
```suggestion
* core: Using different template types when building a HCP Packer
     image iteration will now trigger a runtime build error due to the 
     mixing of template types.
     [GH-1232](https://github.com/hashicorp/packer/pull/12132)
```

**Review by JenGoldstrich — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,71 @@
 ## 1.8.5 (Upcoming)
 
+### NOTES:
+* data/packer-image-iteration has been removed. This was an undocumented and
+     unusable data source that was built for experimentation but not released. It
+     should not affect users in any way but is being mentioned for visibility
+     purposes,  as changes within this release.
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:7]: I think we can drop the `,  as changes within this release.`

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,71 @@
...
+* vsphere@v1.0.7 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-vsphere/releases/tag/v1.0.7)
+* qemu@v1.0.8 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-qemu/releases/tag/v1.0.8)
+
+### IMPROVEMENTS:
+* cmd/hcl2_upgrade: Generate variable block for all referenced user input
+     variables. [GH-12136](https://github.com/hashicorp/packer/pull/12136)
+* cmd/validate: Add support for the `-evaluate-datasources` flag to evaluate
+     the data sources from a template during validation time.
+     [GH-12106](https://github.com/hashicorp/packer/pull/12106)
+     [GH-12106](https://github.com/hashicorp/packer/pull/12106)
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:33]: This link is duplicated

File: CHANGELOG.md (hunk 8 above)
Comment (nywilken) [inline CHANGELOG.md:7]:
```suggestion
     purposes.
```

File: CHANGELOG.md (hunk 9 above)
Comment (nywilken) [inline CHANGELOG.md:33]:
```suggestion
```

Comment (nywilken) [reply CHANGELOG.md:33]: Good catch!

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,70 @@
...
+* core: Bump Go version to 1.18.9 to address vulnerability GO-2022-1144, which
+     concerns the net/http and golang.org/x/net
+     packages.[GH-12153](https://github.com/hashicorp/packer/pull/12153)
+* core: Fix the registration of deleted input artifacts in HCP Packer when
+     setting `keep_input_artifacts` to
+     false.[GH-11462](https://github.com/hashicorp/packer/pull/11967)
+* core: Using different template types when building a HCP Packer
+     image iteration will now trigger a runtime build error due to the 
+     mixing of template types.
+     [GH-1232](https://github.com/hashicorp/packer/pull/12132)
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:66]:
```suggestion
     [GH-12132](https://github.com/hashicorp/packer/pull/12132)
```

File: CHANGELOG.md (hunk 8 above)
Comment (nywilken) [inline CHANGELOG.md:7]:
```suggestion
     purposes.
```

File: CHANGELOG.md (hunk 9 above)
Comment (nywilken) [inline CHANGELOG.md:33]:
```suggestion
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,71 @@
...
+     unusable data source that was built for experimentation but not released. It
+     should not affect users in any way but is being mentioned for visibility
+     purposes,  as changes within this release.
+     [GH-12111](https://github.com/hashicorp/packer/pull/12111)
+
+### FEATURES:
+* core: Metadata for capturing template types such as JSON or HCL2 has been added to the
+     HCP Packer registry metadata. Upon running a `packer build` the type of
+     template used during execution will be sent along to the registry as
+     additional build metadata.[GH-1232](https://github.com/hashicorp/packer/pull/12132)
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:14]:
```suggestion
     additional build metadata.[GH-12132](https://github.com/hashicorp/packer/pull/12132)
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,71 @@
...
+     next available version for a plugin, if the most recent version is not
+     available or has a missing checksum file. This should prevent Packer from
+     trying to install versions that have a GitHub tag but no actual assets.
+     [GH-12103](https://github.com/hashicorp/packer/pull/12103)
+* core: Bump Go version to 1.18.9 to address vulnerability GO-2022-1144, which
+     concerns the net/http and golang.org/x/net
+     packages.[GH-12153](https://github.com/hashicorp/packer/pull/12153)
+* core: Fix the registration of deleted input artifacts in HCP Packer when
+     setting `keep_input_artifacts` to
+     false.[GH-11462](https://github.com/hashicorp/packer/pull/11967)
```
Comment (lbajolet-hashicorp) [inline CHANGELOG.md:63]:
```suggestion
     false.[GH-11967](https://github.com/hashicorp/packer/pull/11967)
```

Comment (nywilken) [reply CHANGELOG.md:62]: Ah thanks!!!!

**Pull request merged**